wagtail_transfer/static_src/components/ContentImportForm/index.js

```diff
@@ -49,11 +49,16 @@ function SubmitButton({ onClick, disabled, numPages }) {
 export default function ImportContentForm({
   localApiBaseUrl,
   sources,
-  onSubmit
+  onSubmit,
+  localCheckUIDUrl
 }) {
   const [source, setSource] = React.useState(null);
   const [sourcePage, setSourcePage] = React.useState(null);
   const [destPage, setDestPage] = React.useState(null);
+  const [
```

Comment:
Might be worth setting this to `null` initially indicating that we don't know if it exists yet. We should hide the choose destination widget until we know that the destination doesn't exist.

Reply:
Yes, it's probably best to separate unknown from false - I've changed this now and made it set alreadyExistsAtDestination to null at first when a new sourcePage is chosen.